metric_config_parser/experiment.py

```diff
@@ -68,6 +77,7 @@ class Experiment:
     reference_branch: Optional[str]
     is_high_population: bool
     app_name: str
+    bucket_config: BucketConfig
```

Comment:
Should this be optional? Or will every experiment have this?

Reply:
BucketConfig is required on the API as well as for the experiment, so if it's not there that should indicate a problem. If you think it would be better, we could deal with this potential error case on the Jetstream side instead and make it Optional here?

Reply:
Recording for posterity -- normandy/v1 experiments do not have `bucket_config`, so we'll make it Optional here and then deal with the requirement for Nimbus experiments in Jetstream.